apps/login/src/utils/session.ts

```diff
@@ -52,9 +58,9 @@ export async function createSessionAndUpdateCookie(
         const sessionCookie: CustomCookieData = {
           id: createdSession.sessionId,
           token: createdSession.sessionToken,
-          creationDate: `${response.session.creationDate?.toDate().getTime() ?? ""}`,
-          expirationDate: `${response.session.expirationDate?.toDate().getTime() ?? ""}`,
-          changeDate: `${response.session.changeDate?.toDate().getTime() ?? ""}`,
+          creationDate: `${toDate(response.session.creationDate)?.getTime() ?? ""}`,
```

Comment:
**Important:** use `toDate` to convert to `Date` when the value may be `undefined`